packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.tsx

```diff
@@ -525,6 +527,7 @@ export class UnthemedSpanBarRow extends React.PureComponent<SpanBarRowProps> {
             [styles.viewExpandedAndMatchingFilter]: isMatchingFilter && isDetailExpanded,
           })}
           data-test-id="span-view"
+          data-testid="span-view"
```

Comment:
this is the correct syntax, so i'd replace the `data-test-id` declaration completely and fix any other tests that fail. you should be able to just globally find `data-test-id="span-view"` and replace with `data-testid="span-view"`